lib/elixir/pages/Compatibility and Deprecations.md

```diff
@@ -1,16 +1,56 @@
-# Deprecations
+# Compatibility and Deprecations
 
-## Policy
+## Compatibility between Elixir and Erlang/OTP
+
+Erlang/OTP versioning is independent from the versioning of Elixir. Each version of Elixir supports a specific range of Erlang/OTP versions. The compatibility table is show below.
+
+### Table of Erlang/OTP compatibility
+
+Elixir version | Minimum required Erlang/OTP version | Maximum supported Erlang/OTP version
```

Comment:
Would it look better to write?

```
Elixir version | Supported Erlang/OTP versions
--------- | -----
v1.5.2 | 18-20
```

Reply:
It looks better so I pushed a commit that uses that style. However I wonder if we can improve this even more because this way some lines will have just one number (otherwise we end up with `17 - 17`).